subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -687,6 +693,11 @@ void ull_conn_iso_ticker_cb(uint32_t ticks_at_expire, uint32_t ticks_drift,
 		if (cis->lll.handle != 0xFFFF && cis->lll.active) {
 			cis->lll.event_count += (lazy + 1U);
 
+#if !defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
+			cis->lll.event_count -= cis->lll.lazy_active;
+			cis->lll.lazy_active = 0U;
```

Comment:
So `lazy_active` is a number of events?

Reply:
yes, updated the comment in header file.